app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments, foreign_key: 'user_id'
+  has_many :likes, foreign_key: 'user_id'
```

Comment:
- [ ] As indicated in the ERD image shared with you, the foreign_key should be `author_id`, not `user_id`.

  Kindly change these. Also, ensure these changes are reflected in the respective database tables

  ![sadiq-author](https://user-images.githubusercontent.com/36542986/162934792-081abebf-fa47-4177-b787-955dfec94e01.jpg)